### ISSUE TITLE
Entities from external assemblies are not navigable in tooltips

### ISSUE THREAD

**Issue by kerams** [dotnet/fsharp#16196]
It's not possible to click on types, methods, properties, etc. from within tooltips in order to navigate to external metadata.

![image](https://github.com/dotnet/fsharp/assets/5063478/afcc5569-ef09-4703-a8ab-c97f4d16100b)

**Describe alternatives you've considered**

Type out the entity I want to access and F12 on that :).

**Additional context**

VS 17.8.0 Preview 3.0

**Comment by kerams** [comment]
Was about to submit a feature request when I found this ;D

Anyway, here's a comparison with our older sibling

https://github.com/user-attachments/assets/3569732e-f051-44a6-8817-29bb5a010de4

(Bizarrely, `'TValue` in this example is navigable. Not sure if it's a bug or what the intent was - it just goes to `System`.)

**Comment by majocha** [comment]
Yeah, the implementation of this feature is old and rather simplistic. It just attaches whatever compiler range is available in the AST during tooltip layout. It should probably also compute XmlDocSig for external symbols.
